int64 robot_id
geometry_msgs/Point[] points
